src/pages/mine/index.tsx

```diff
@@ -57,9 +57,9 @@ const Content = () => {
 
     if (isLoading)
         return (
-            <div className={styles.tasksContainer}>
+            <div className={styles.mineTasksContainer}>
```

Comment:
Did we already have this class? I can't find it defined in the new code.

Reply:
it is already there.